service/src/main/java/tds/assessment/repositories/impl/ItemQueryRepositoryImpl.java

```diff
@@ -93,7 +93,7 @@ public ItemQueryRepositoryImpl(final NamedParameterJdbcTemplate jdbcTemplate) {
                 "AND \n" +
                 "   adminItems.isactive = 1 \n" +
                 "AND \n" +
-                "   formItem.isactive = 1";
+                "   (formItem.isactive = 1 OR formItem.isactive IS NULL)";
```

Comment:
The `LEFT JOIN` totally makes sense, but this is a bit confusing to me.  Is the idea here that if we _are_ looking for forms that we get the active and inactive (or whatever a `NULL isactive` field means) forms and let the caller decide what to do with the result?  I'm sure it'll work fine, just seems like an odd condition to me.